### PR TITLE
AT-37 Creating student page on frontend

### PR DESCRIPTION
**Pull request by RyanVanDijck**

### CODE REVIEW

**Review by dearshrewdwit — CHANGES_REQUESTED** [review]
Make sure you run `CI=true npm run build` locally and fix any errors - currently the PR is not passing CI tests

I'll review when it passes 👍

File: src/App.js [hunk 1]
```diff
@@ -34,7 +35,11 @@ function App() {
             <Route path="/students">
               <p>replace me</p>
             </Route>
-            
+            </Switch>
+            <Switch>
+              <Route exact path='/student/:id'>
```
Comment (dearshrewdwit) [inline src/App.js:40]: Resources are almost always plural in URLs:  should be `/students/:id`

File: src/components/Students/StudentInfo.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    constructor(props) {
+        super(props)
+        this.state = {
+            id: props.id,
+            studentData: {} 
+        }
+      }
+
+    async componentDidMount(){
+        await fetch(`${process.env.REACT_APP_API_URL}/api/students:${this.state.id}`)
```
Comment (dearshrewdwit) [inline src/components/Students/StudentInfo.js:19]: Is this URL formed correctly - should it not be `/api/students/{this.state.id}` ?

Check your endpoint :)

File: src/components/Students/StudentInfo.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+const GetStudentInfo = (props) => {
+    return (<StudentInfo id={useParams().id}/>)
+}
+
+class StudentInfo extends Component{
+
+    constructor(props) {
+        super(props)
+        this.state = {
+            id: props.id,
```
Comment (dearshrewdwit) [inline src/components/Students/StudentInfo.js:13]: Do you need to set state for `id`?  Could you leave it as a property, and access it via `this.props.id`?

**Review by dearshrewdwit — CHANGES_REQUESTED** [review]

File: src/components/Students/StudentInfo.js (hunk 3 above)
Comment (RyanVanDijck) [reply src/components/Students/StudentInfo.js]: Ah, this was left over from the old solution. 

Fixed here:
https://github.com/digital-futures-academy/atlas-frontend/pull/9/commits/a9035ceaf3fa03ab9685a5ebeeb6643f125c7600

File: src/components/Students/StudentInfo.js (hunk 2 above)
Comment (RyanVanDijck) [reply src/components/Students/StudentInfo.js]: Fixed here:

https://github.com/digital-futures-academy/atlas-frontend/pull/9/commits/a9035ceaf3fa03ab9685a5ebeeb6643f125c7600

File: src/App.js (hunk 1 above)
Comment (RyanVanDijck) [reply src/App.js]: Fixed here:
https://github.com/digital-futures-academy/atlas-frontend/pull/9/commits/2777305e92270cde87d39a14d5e941b6b0c89e9a

**Review by dearshrewdwit — APPROVED** [review]

**Review by bferenczy0 — APPROVED** [review]

**Pull request merged**